systemd/systemd.go

```diff
@@ -283,6 +283,9 @@ var jctl = func(svcs []string, n int, follow bool) (io.ReadCloser, error) {
 	if follow {
 		args = append(args, "-f") // ... + 1 == 6
 	}
+	if len(namespace) > 0 {
```

Comment:
(nitpick^9) given that this is a string, it feels slightly nicer to use: `if namespace != "" {` but I understand this is super nitpicky.

Reply:
Good suggestion, thanks!